modules/apps/transfer/keeper/relay_test.go

```diff
@@ -169,21 +169,6 @@ func (suite *KeeperTestSuite) TestSendTransfer() {
 		// 	},
 		// 	channeltypes.ErrInvalidPacket,
 		// },
-		{
```

Comment:
This test now doesn't succeed due to the following logic:

- `Transfer` does not check for the version/forwarding combination anymore
- the function that creates the packet data will ignore everything in forwarding when the version is V1.
Note that there is still a check on `OnSendPacket` but it will not be triggered because Hops will not be present.


I'll open an issue to add tests specifically for `OnSendPacket` since we currently do not have them

Reply:
It should? I wonder if this was not working because of the `1` -> `0` change?

Reply:
ouch thanks, yeah it does. I'm trying to give up coffee starting today and you can see the results :(